src/git.zsh

```diff
@@ -170,6 +137,40 @@ _fzf_complete_git() {
     _fzf_path_completion "$prefix" $@
 }
 
+_fzf_complete_git_prefix() {
+    if [[ -z ${prefix:/--[^-]##*=*} ]]; then
+        echo ${prefix/=*/=}
+    fi
+}
+
+_fzf_complete_git_has_options() {
+    local option
+    local options=$1
+    local last_argument=$2
+    local prefix=$3
+
+    for option in ${(z)options}; do
+        if [[ ${#option} = 1 ]]; then
+            if [[ $last_argument =~ "^-[^-]*$option" ]]; then
+                echo 0
```

Comment:
I'd like this function to express the result by its status code. Why did it choose to use stdout for that purpose?

Reply:
When I used status code as below, the tests was failed because zunit sets ERR_EXIT.
```
_fzf_complete_git_has_options 'm message' $last_argument $prefix
if [[ $? = 0 ]]; then
fi
```
I came up with an idea, so I will try it.

Reply:
Just use if clause.